### PR TITLE
feat: add Sepolia and Arbitrum Sepolia networks

### PR DESCRIPTION
**Pull request by bartlomiej94**

### CODE REVIEW

File: src/lib/dataEntities/networks.ts [hunk 1]
```diff
@@ -246,6 +255,42 @@ export const l2Networks: L2Networks = {
...
+      l2ERC20Gateway: '0x2eC7Bc552CE8E51f098325D2FcF0d3b9d3d2A9a2',
+      l2GatewayRouter: '0xE5B9d8d42d656d1DcB8065A6c012FE3780246041',
+      l2Multicall: '0x108B25170319f38DbED14cA9716C54E5D1FF4623',
+      l2ProxyAdmin: '0xeC377B42712608B0356CC54Da81B2be1A4982bAb',
+      l2Weth: '0xe39Ab88f8A4777030A534146A9Ca3B52bd5D43A3',
+      l2WethGateway: '0xf9F2e89c8347BD96742Cc07095dee490e64301d6',
+    },
+    nitroGenesisBlock: 0,
+    nitroGenesisL1Block: 0,
+    depositTimeout: 3960000,
```
Comment (bartlomiej94) [inline src/lib/dataEntities/networks.ts:292]: correct?

File: src/lib/dataEntities/networks.ts [hunk 2]
```diff
@@ -142,6 +142,15 @@ export const l1Networks: L1Networks = {
     partnerChainIDs: [421613],
     isArbitrum: false,
   },
+  11155111: {
+    chainID: 11155111,
+    name: 'Sepolia',
+    explorerUrl: 'https://sepolia.etherscan.io',
+    partnerChainIDs: [421416],
+    blockTime: 15,
```
[the remark this answers is not in the document]
Comment (gzeoneth) [reply src/lib/dataEntities/networks.ts:150]:
```suggestion
    blockTime: 12,
```

File: src/lib/dataEntities/networks.ts [hunk 3]
```diff
@@ -246,6 +255,42 @@ export const l2Networks: L2Networks = {
      */
     depositTimeout: 1800000,
   },
+  421416: {
```
Comment (chrstph-dvx) [inline src/lib/dataEntities/networks.ts:258]: Should be 42141**4** no?

https://sepolia.etherscan.io/address/0x13C75Ec6EEBd45837C53D3A91561D88C296fC5A5#readProxyContract (under ChainId)

Comment (gzeoneth) [inline src/lib/dataEntities/networks.ts:258]:
```suggestion
  421614: {
```

File: src/lib/dataEntities/networks.ts [hunk 4]
```diff
@@ -246,6 +255,42 @@ export const l2Networks: L2Networks = {
...
+    chainID: 421416,
+    confirmPeriodBlocks: 20,
```
[the remark this answers is not in the document]
Comment (gzeoneth) [reply src/lib/dataEntities/networks.ts:260]: correct

File: src/lib/dataEntities/networks.ts [hunk 5]
```diff
@@ -142,6 +142,15 @@ export const l1Networks: L1Networks = {
     partnerChainIDs: [421613],
     isArbitrum: false,
   },
+  11155111: {
+    chainID: 11155111,
+    name: 'Sepolia',
+    explorerUrl: 'https://sepolia.etherscan.io',
+    partnerChainIDs: [421416],
```
Comment (gzeoneth) [inline src/lib/dataEntities/networks.ts:149]:
```suggestion
    partnerChainIDs: [421614],
```

File: src/lib/dataEntities/networks.ts (hunk 1 above)
Comment (gzeoneth) [reply src/lib/dataEntities/networks.ts:292]: maybe
```suggestion
    depositTimeout: 1800000,
```

File: src/lib/dataEntities/networks.ts [hunk 6]
```diff
@@ -246,6 +255,42 @@ export const l2Networks: L2Networks = {
      */
     depositTimeout: 1800000,
   },
+  421416: {
+    chainID: 421416,
```
Comment (gzeoneth) [inline src/lib/dataEntities/networks.ts:259]:
```suggestion
    chainID: 421614,
```

File: src/lib/dataEntities/networks.ts (hunk 3 above)
Comment (gzeoneth) [reply src/lib/dataEntities/networks.ts:258]: lol you also got it wrong its 421614 not 421416 or 421414

File: src/lib/dataEntities/networks.ts [hunk 7]
```diff
@@ -246,6 +255,42 @@ export const l2Networks: L2Networks = {
...
+      bridge: '0x3f24803B4a16b61d3Fc122E4728aEF2Ef93DeBd8',
+      inbox: '0xA92Ed990585e4F8656E2aC0E33cA8FDe5473CB92',
+      outbox: '0x9AABB8145880dFf3071C1A62681E19E2D2E1141B',
+      rollup: '0x13C75Ec6EEBd45837C53D3A91561D88C296fC5A5',
+      sequencerInbox: '0xf763BdeAbA8aDD268A0A88F74231118e05b4A872',
+    },
+    explorerUrl: 'https://sepolia-explorer.arbitrum.io',
+    isArbitrum: true,
+    isCustom: false,
+    name: 'Arbitrum Sepolia',
```
Comment (gzeoneth) [inline src/lib/dataEntities/networks.ts:271]:
```suggestion
    name: 'Arbitrum Rollup Sepolia Testnet',
```

**Pull request merged**